CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,896 @@
+be specified to allow for this functionality in the future, though.
+
+## User research
+
+Use cases in this explainer were collected from the discussion in issue
+[8930](https://github.com/w3c/csswg-drafts/issues/8930). Additional inspiration
+was drawn from discussions in issues
+[9447](https://github.com/w3c/csswg-drafts/issues/9447), and OpenUI issue
+[730](https://github.com/openui/open-ui/issues/730).
+
+There are also many articules on the web that walk authors through various
+options for creating custom tooltips using tools available to them today.
+One such [article](https://blog.replaybird.com/css-tooltip-examples/)
+walks through different options, along with various examples and use cases
+for tooltips on the web.
+
+## Existing and upcoming tools available for custom tooltips
+
+If an author would like to adjust the style of their tooltip today, they must resort
+to the one of the following alternative approaches to the built-in tooltip that they
+get with the `title` attribute.
+
+### Use an existing library or component
+
+Authors can use an existing library or component for a fully customizable tooltip.
+Some example libraries include:
+  - https://fluent2.microsoft.design/components/web/react/core/tooltip/usage
+  - https://www.opentip.org/
+  - https://floating-ui.com/?utm_source=popper.js.org
+  - https://www.npmjs.com/search?ranking=popularity&q=tooltip
+  - Many more   
+
+It is clear that there are many libraries and components for creating custom tooltips
+that authors can use to solve their need for more customization.
+
+If the library chosen if well architected to properly handle accessibility, this can
+also be a convenient way to get such functionality "for free". However, relying on
+external library to get accessibility right can be risky, which may limit the usability
+if not done correctly.
+
+The other downside of this approach is that it requires heavy use of JS, which can be
+slower, and shouldn't be necessary if all you want to do is adjust some basic styles.
+
+### Create a fully custom tooltip
+
+Authors can also choose to create their own custom tooltip or component, which
+requires their own input management, proper accessibility support and proper
+handling of the tooltip positioning, which can be complex.
+
+This does allow authors to acheive their goal of full cusomizability, but it often
+requires heavy use of JS, or use of `:before` or `:after` pseudos, which can make
+getting accessibility correct quite difficult.
+
+As such, this solution requires a lot of work to get right, especially if all they
+want to do is adjust a few simple styles on the built-in tooltip.
+
+#### Utilize `popover`, CSS anchor positioning, and/or `interesttarget`
+
+Although creating a custom tooltip can be cumbersome for an author who is looking
+to just adjust a few simple styles, they do have some new web APIs, and APIs that
+are in the works, that can help them in accomplishing a fully customizable tooltip
```

Comment:
"they do have some new web APIs, and APIs that are in the works" - maybe say "they do have some new (and upcoming) web APIs""

Reply:
Done